lms/djangoapps/instructor/static/instructor/ProblemBrowser/data/actions/constants.js

```diff
@@ -0,0 +1,4 @@
+export const REPORT_GENERATION_REQUEST = 'PROBLEM_RESPONSES_REQUEST';
+export const REPORT_GENERATION_SUCCESS = 'PROBLEM_RESPONSES_SUCCESS';
+export const REPORT_GENERATION_ERROR = 'PROBLEM_RESPONSES_ERROR';
+export const REPORT_GENERATION_TIMEOUT = 'PROBLEM_RESPONSES_TIMEOUT';
```

Comment:
Please add a comment explaining what `REPORT_GENERATION_TIMEOUT` means - does it mean the report timed out and failed, or that we're due to poll the status again on a fixed interval?

Reply:
I've renamed it to be clearer.